pkg/zcache/config.go

```diff
@@ -56,7 +56,6 @@ func (c *LocalConfig) ToBigCacheConfig() bigcache.Config {
 type CombinedConfig struct {
 	Local              *LocalConfig
 	Remote             *RemoteConfig
-	GlobalLogger       *zap.Logger
```

Comment:
This was there because we enable or disable the logging from indexing (through the parser). Is it possible to disable logging now without this?

Reply:
fixed